fabfile.py

```diff
@@ -102,7 +106,7 @@ def production():
     """
     Select production instance(s)
     """
-    env.hosts = ['cabot.arachnys.com']
```

Comment:
@dukelion not sure you want this in here (although silly to have Arachnys hostname too)